src/anemoi/graphs/generate/hexagonal.py

```diff
@@ -121,6 +114,8 @@ def add_edges_to_nx_graph(
     depth_children : int
         The number of resolution levels to consider for the connections of children. Defaults to 1, which includes
```

Comment:
Default has changed?

Reply:
With x_hops=1, children of the same node will be connected. So, depth_children=0 & 1 will produce the same graph.